priv/repo/migrations/20170427090331_create_class.exs

```diff
@@ -0,0 +1,18 @@
+defmodule CoursePlanner.Repo.Migrations.CreateClass do
+  use Ecto.Migration
+
+  def change do
+    create table(:classes) do
+      add :class_date, :date
+      add :starting_at, :time
+      add :finishes_at, :time
+      add :status, :entity_status
+      add :deleted_at, :naive_datetime
+      add :course_id, references(:courses, on_delete: :delete_all)
```

Comment:
I think that's fine as a database constraint, if we do happen to hard delete a course we don't leave orphan class records